src/charts/c3/c3-chart-defaults.js

```diff
@@ -37,5 +37,90 @@ angular.module('patternfly.charts').constant('c3ChartDefaults', {
       color: this.getDefaultDonutColor(),
       tooltip: this.getDefaultDonutTooltip()
     };
+  },
+
+  /*
+   * Sparkline Defaults
+   */
+
+  getDefaultSparklineArea: function () {
+    return {
+      zerobased: true
+    };
+  },
+
+  getDefaultSparklineSize: function () {
+    return {
+      height: 60
+    };
+  },
+
+  getDefaultSparklineAxis: function () {
+    return {
+      x: {
+        show: false
+      },
+      y: {
+        show: false
+      }
+    };
+  },
+
+  getDefaultSparklineColor: function () {
+    return {
+      pattern: ['#0088ce', '#00659c', '#3f9c35', '#ec7a08', '#cc0000']
+    };
+  },
+
+  getDefaultSparklineLegend: function () {
+    return {
+      show: false
+    };
+  },
+
+  getDefaultSparklinePoint: function () {
+    return {
+      r: 1,
+      focus: {
+        expand: {
+          r: 4
+        }
+      }
+    };
+  },
+
+  getDefaultSparklineTooltip: function () {
+    return {
+      // because a sparkline should only contain a single data column, the tooltip will only work for a single data column
+      contents: function (d) {
+        return '<span class="c3-tooltip-sparkline">' + d[0].value + ' ' + d[0].name + '</span>';
+      }
+    };
+  },
+
+  getDefaultSparklineConfig: function () {
+    return {
+      area: this.getDefaultSparklineArea(),
+      size: this.getDefaultSparklineSize(),
+      axis: this.getDefaultSparklineAxis(),
+      color: this.getDefaultSparklineColor(),
+      legend: this.getDefaultSparklineLegend(),
+      point: this.getDefaultSparklinePoint(),
+      tooltip: this.getDefaultSparklineTooltip()
+    };
+  },
+
+  /*
+   * Tooltips
+   */
+  getTooltipTableHTML: function (tipRows) {
+    return '<div class="module-triangle-bottom">' +
+    '  <table class="c3-tooltip">' +
+    '    <tbody>' +
+           tipRows +
+    '    </tbody>' +
+    '  </table>' +
+    '</div>';
```

Comment:
I personally think this should stay in the directive as that is really the only place you should have HTML (other than .html files) in Angular.